theano/gpuarray/extra_ops.py

```diff
@@ -40,7 +39,6 @@ def get_params(self, node):
 
     def make_node(self, x):
         assert x.type.dtype == 'float32', "Only float32 supported for GpuCumSum"
-        x = as_gpuarray_variable(x, infer_context_name(x))
```

Comment:
If it is not needed (like for instance for GpuAllocEmpty, which does not have any way of inferring a context), I don't know what the best practice would be. @abergeron what do you think?